sequencer/src/lib.rs

```diff
@@ -380,20 +397,46 @@ pub async fn init_node<Ver: StaticVersionType + 'static>(
     };
     let node_index = config.node_index;
 
-    // Initialize networking.
+    // Initialize the push CDN network (and perform the initial connection)
+    let cdn_network = PushCdnNetwork::new(
+        network_params.cdn_endpoint,
+        vec!["Global".into(), "DA".into()],
+        KeyPair {
+            public_key: WrappedSignatureKey(my_config.public_key),
+            private_key: my_config.private_key.clone(),
+        },
+    )
+    .await
+    .with_context(|| "Failed to create CDN network")?;
```

Comment:
I have in-progress CDN changes to make the connection be lazily initialized, but they aren't in yet.